src/components/ApprovalRequest/ApprovalRequest.tsx

```diff
@@ -112,7 +113,7 @@ const ApprovalRequest: FC<ApprovalRequestProps> = ({
   return (
     <>
       <BackButton onClick={(): void => setStage(StageState.VIEW)} theme="blue" />
-      <div className={styles.card}>
+      <Card className={styles.approvalRequestCard}>
```

Comment:
No chance we can put the back button in the card also? And the logout one?

Reply:
I tried it out but it looked a bit weird, think we may have to rethink a bit more than just plopping it in the card